<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendManage</class>
 <widget class="QDialog" name="friendManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看好友信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>214</width>
     <height>255</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_acc">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_name">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_email">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>电话号码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_tel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>头像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_img">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>删除好友</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Quit">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
